portal-settlement/templates/secrets.yaml

```diff
@@ -1,14 +1,14 @@
 {{- $dbHost := (.Values.db.host | replace "$release_name" .Release.Name) }}
-
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "settlement.fullname" . }}
+  name: {{ template "portal-settlement.fullname" . }}
```

Comment:
Would it be possible to use consistent naming with regard to portals?

I see that we have a `finance-portal`, then why not a `settlement-portal`?

Reply:
although I agree with consistent naming, `settlement-portal` maybe sounds like another portal. the service is a complement for the finance-portal, so perhaps `finance-portal-settlement-management` is a better name.
Please let me know what you think.